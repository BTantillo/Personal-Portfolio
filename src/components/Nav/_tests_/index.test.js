import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Navbar from '..';

afterEach(cleanup);

describe('Nav component', () => {
    // baseline to verify that the component is rendering
    it('renders', () => {
        render(<Navbar />);
    });
    // snapshot test
    it('matches snapshot', () => {
        const { asFragment } = render(<Navbar />);
        expect(asFragment()).toMatchSnapshot();});
})