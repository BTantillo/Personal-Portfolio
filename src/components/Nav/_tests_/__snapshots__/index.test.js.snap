// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot 1`] = `
<DocumentFragment>
  <header>
    <h2>
      <a
        href="/"
      >
        <span
          aria-label="waving hand"
          role="img"
        >
           ðŸ‘‹ 
        </span>
         It's Will!
      </a>
    </h2>
    <nav>
      <ul
        class="flex-row"
      >
        <li
          class="mx-2"
        >
          <a
            href="#about"
          >
            About me
          </a>
        </li>
        <li>
          <span>
            Contact
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            front-end
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            back-end
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            social-media
          </span>
        </li>
        <li
          class="mx-1"
        >
          <span>
            podcasting
          </span>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
